lmdeploy/pytorch/models/deepseek.py

```diff
@@ -47,11 +47,12 @@ def _contiguous_batching_forward_impl(
         support.
         """
         context = self.context.context
-        history_lengths = context.history_lengths
         kv_seq_length = context.kv_seq_length
         q_seq_length = context.q_seq_length
         q_start_loc = context.q_start_loc
         block_offsets = context.block_offsets
+        max_q_seq_length = context.max_q_seq_length
```

Comment:
somthing wrong with deepseek models
```
lmdeploy chat torch /share/140/deepseek/deepseek-coder-33b-instruct/
2024-03-11 18:39:49,722 - lmdeploy - INFO - Checking environment for PyTorch Engine.
2024-03-11 18:39:49,912 - lmdeploy - INFO - Checking model.
Loading checkpoint shards:   0%|                                                                                        | 0/7 [00:00<?, ?it/s]/opt/py38/lib/python3.8/site-packages/torch/_utils.py:831: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()
  return self.fget.__get__(instance, owner)()
Loading checkpoint shards: 100%|████████████████████████████████████████████████████████████████████████████████| 7/7 [00:34<00:00,  4.88s/it]
2024-03-11 18:41:09,359 - lmdeploy - INFO - build CacheEngine with config:CacheConfig(block_size=64, num_cpu_blocks=264, num_gpu_blocks=758, window_size=-1, cache_max_entry_count=0.8)
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
match template: <Mistral-7B-Instruct>

double enter to end input >>> hello

<s>[INST] hello [/INST] huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
        - Avoid using `tokenizers` before the fork if possible
        - Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
        - Avoid using `tokenizers` before the fork if possible
        - Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
        - Avoid using `tokenizers` before the fork if possible
        - Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
        - Avoid using `tokenizers` before the fork if possible
        - Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
</s>
<s>[INST] hello [/INST]</s>
<s>[INST] hello [/INST]</s>
<s>[INST] hello [/INST]huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
        - Avoid using `tokenizers` before the fork if possible
        - Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
</s>
<s>[INST] hello [/INST]</s>
<s>[INST] hello [/INST]</s>
<s>[INST] hello [/INST]</s>
<s>[INST] hello [/INST]</s>
<s>[INST] hello [/INST]</s>
<s>[INST] hello [/INST]</s>
<s>[INST] hello [/INST]</s>
<s>[INST] hello [/INST]</s>
<s>[INST] hello [/INST]</s>
<s>[INST] hello [/INST]</s>
<s>[INST] hello [/INST]</s> ...`